### ISSUE TITLE
Add Rollup as the module bundler.

### ISSUE THREAD

**Issue by rafael-lima** [pagarme/pagarme-js#38]
I watched the talk on pagarme channel about your JS library, and one of the improvement points mentioned was the library size. So, I saw that you using webpack, and the webpack bundling generated is too big, thus I thought you might use the Rollup as the module bundler, at least partially, because the Rollup generates a really clean and smaller bundling, and I think that will reduce a good part of the size.

**Comment by rafael-lima** [comment]
https://medium.com/webpack/webpack-and-rollup-the-same-but-different-a41ad427058c

**Comment by derekstavis** [comment]
Hi @rafael-lima, thanks for taking time to open this discussion 😄 

Currently, the main issue we have with the library size isn't caused by the module bundler, but by dependencies we have been using to do encryption/signing and http communication.

As we are pleased by standards, we used an implementation of `fetch` standard named `isomorphic-fetch`. It happens that this library depends on a lot of other packages, including encoding packages and others. I have opened issue #35 to address this with the idea of using Node.js' http client, which webpack should compile to browserify's http package when target is web.

About a Rollup migration, I have read the article and the reasons are pretty good, primarily in the context of React, a big library which have lots of modules in its own codebase -- Loading every module by evaluating a function is both time and space consuming, something that may not be true in a small library as `pagarme.-js`.

Although we have been using Webpack as our module bundler throughout our projects, we would be happy to see how Rollup would make`pagarme-js` faster to load and how much it would decrease our bundle size. That said, a Pull Request migrating Webpack to Rollup would be nice to allow us to analyze how they compare in the context of this project!
